adding_metadata_sql.md

```diff
@@ -0,0 +1,193 @@
+
+1. correcting systemic data errors involving large numbers of rows
+2. assigning non-point geometries to sites [this might be possible in YABA, I don't know]
+3. associating experiments with sites and treatments
+4. associating sites with cultivars
+
+### Metadata Entry Workflow for BETYdb
+
+The following steps have been implemented in the BETYdb-YABA API and python client, but are described below for reference
+
+Here we show SQL code used to add metadata required each season by the TERRA REF database, including experiments, sites, treatments, cultivars, and citations that is required prior to uploading the trait or yield data. Most PEcAn users _will not_ need to add experiments and cultivars, or associate these records with sites.
+
+Also note that in the TERRA REF and other agronomic applications, each record in the 'sites' table will correspond to an experimental plot.
+
+### Step 1: Add new experiments
+
+```sql
+insert into experiments (name, start_date, end_date, user_id) 
+values ('MAC Season 6: Sorghum BAP', '2018-04-06', '2018-08-01', 'some text', 'some text', 6000000004);
+```
+
+### Step 2: Add new sites
+
+_must provide shapefile for a site to get geometry_
+
+```sql
+insert into sites (city, state, country, sitename) 
+values ('Maricopa', 'Arizona', 'USA', 'MAC Field Scanner Season 7 Range 9 Column 15');
+```
+
+For the TERRA REF Project, plot definitions may be copied from previous season if same plots are used.
+
+```sql
+with season6 as (
+  select city, state, replace(sitename, 'Season 4', 'Season 6') as sitename, greenhouse, geometry, time_zone from sites where sitename like '%Season 4%' 
+)
+insert into sites (city, state, sitename, greenhouse, geometry, time_zone) select * from season6;
+```
+
+
+### Step 3: Add new treatments
+
+```sql
+insert into treatments (name, definition, control) 
+values ('MAC Season 6: Sorghum', 'some text', 't');
+```
+
+### Step 4: Add new cultivars
+
+_each cultivar should be associated with a specie_
+
+_new species must be added to species table before referencing_
+
+```sql
+insert into cultivars (name, specie_id) 
+values ('RIL-CS27_(TX2910/(Macia/R07007)-CS44)-CSF1-PRF2-CS27', 2588);
+```
+
+### Step 5: Add new citations
+
+```sql
+insert into citations (author, year, title) 
+values ('Newcomb, Maria', 2016, 'Maricopa Agricultural Center Field Activities');
+```
+
+### Step 6: Associate experiments with sites
+
+```sql
+insert into experiments_sites (experiment_id, site_id) values 
+  ((select id from experiments where name = 'MAC Season 6: Sorghum BAP'),
+  (select id from sites where sitename = 'MAC Field Scanner Season 6 Range 1 Column 1 E'));
+```
+
+When adding a new season for the TERRA REF project, a statement like the following can be used for associating experiments and sites since MAC Field Center sites are consistently named following the format `MAC Field Scanner Season x Range a Column b`.
```

Comment:
```suggestion
When adding a new season for the TERRA REF project, a statement like the following can be used for associating the new season's experiment [or "an experiment for the new season"] with all of the new season's sites. For example, since MAC Field Center sites are consistently named following the format MAC Field Scanner Season x Range a Column b, we could use the following statement to associate the experment named "MAC Season 6: Sorghum BAP" with all Season 6 sites:
```